README.md

```diff
@@ -256,41 +269,110 @@ scripts/build.sh
 
 `git clone https://github.com/fabric8io/fabric8-online.git`
 
-* Set some environment variables (or run `. ./scripts/setenv-for-deploy.sh`)
+* You may need to override the full path to the `fabric8-online` binary (provided by the `FABRIC8_ONLINE_PATH`
+environment variable), according to where you cloned the `fabric8-online` git repository :
+
+```bash
+export FABRIC8_ONLINE_PATH="<path where you cloned it>/fabric8-online/"
+```
+
+For all the scripts that deploy or rollout to Minishift, you should have minishift running.
+This can be done with the following command:
 
 ```bash
-export OPENSHIFT_USERNAME="developer"
-export OPENSHIFT_PASSWORD="developer"
-export CHE_OPENSHIFT_PROJECT="eclipse-che"
-export OPENSHIFT_NAMESPACE_URL="${CHE_OPENSHIFT_PROJECT}.$(minishift ip).nip.io"
-export CHE_LOG_LEVEL="INFO"
-export CHE_DEBUGGING_ENABLED="false"
-export FABRIC8_ONLINE_PATH="/home/user/github/fabric8-online/"
+minishift start
 ```
 
-* OPENSHIFT_USERNAME and OPENSHIFT_PASSWORD are used to configure which Openshift account will be used
-* CHE_OPENSHIFT_PROJECT is the name of the project in Openshift in which Che will be deployed
-* OPENSHIFT_NAMESPACE_URL is the base URL used for all Che routes. Once deployed Che dashboard will be available at http://che-${OPENSHIFT_NAMESPACE_URL}.
-* CHE_LOG_LEVEL is the logging level (DEBUG, INFO, WARN, ERROR etc)
-* CHE_DEBUGGING_ENABLED set this to "true" to allow remote Java debugging of the Che server
-* FABRIC8_ONLINE_PATH is the full path to the fabric8-online binary
+### RedHat distribution build scripts
+
+This assumes that the upstream Che has been previously fully built.
+
+In the `rh-che` repository, the RedHat distribution can be built with one of the following scripts:
+
+- `dev-scripts/build_fabric8.sh`:
+    - runs the RH maven build with the options passed as arguments,
+    - generates the docker images for the upstream and the RH
+distribution and tag them appropriately in the *local docker
+environment*.
+
+- `dev-scripts/build_fabric8_and_deploy.sh`:
+    - runs the RH maven build with the options passed as arguments,
+    - generates the docker images for the upstream and the RH
+distribution and tag them appropriately in the *minishift docker
+environment*
```

Comment:
you can summarize the first two points with just
- runs `dev-scripts/build_fabric8.sh`

I would not mention "in the local (or minishift) docker environment" because you really don't know if the user has previously run `eval $(minishift docker-env)`.